src/task/args.rs

```diff
@@ -75,10 +75,23 @@ pub(super) fn read() -> ArgMatches {
                         .short('q')
                         .action(ArgAction::SetTrue)
                         .help("do not print table and cluster yaml"),
-                    Arg::new("export-state")
-                        .long("export-state")
+                    Arg::new("state-dir")
+                        .long("state-dir")
+                        .env("GENIN_STATE_DIR")
                         .action(ArgAction::Set)
-                        .help("export the build state"),
+                        .help("override .geninstate directory location"),
+                    Arg::new("recreate")
+                        .long("recreate")
+                        .action(ArgAction::SetTrue)
+                        .help(
+                            "Using the genin configuration and the inventory to be \
+                        modified creates a new inventory",
+                        ),
```

Comment:
Не очень понятен смысл на мой взгляд, может так:

```suggestion
                        .help(
                            "Delete the existing state before building",
                        ),
```

Вообще есть у меня еще мысль, что рекрейт не должен удалять сохраненное состояние, а можт лучше архивировать его и перемещать в рядом лежащую спец.директорию. Просто каков шанс, что человек не прочитает смысл флага и просто незачем дропнет весь свой стейт заодно